src/components/table-transactions/table-transactions.js

```diff
@@ -59,11 +78,12 @@ export class TableTransactionsComponent {
   }
 
   handleDeleteClick(e) {
-    this.delTransaction(e.target.closest(".table-transactions__tr"));
+    this.confirmDialog.showDialog(e.target.closest(".table-transactions__tr"));
```

Comment:
you do not need to pass element to dialog -- you can just temporally save this to component's instance, so you will be able to access it in `handleDeleteConfirm` like this `this.elemToDelete = e.target.closest(".table-transactions__tr")` -- and then you will clear it after deleting element